src/Response.php

```diff
@@ -93,6 +93,10 @@ public function __construct(
         $version = '1.1',
         $reason = null
     ) {
+        if (!is_int($status)) {
```

Comment:
this is breaking for people that used numeric values like `'200'` (by accident). So please also accept those values. When requiring php 7, we can just switch this to a type declaration and then it will be casted automatically (or cause a failure in strict mode).

Reply:
> The response object could validate the status code and accept only numeric forms that are `integerish` and throw exception in any other case.

So just to be clear.. you are in favor of the solution above.. did I understood correct?

Reply:
Yes. Just `if (!is_numeric($status)) {` should be good enough.

Reply:
basically it is not enough.. what happens to strings like '1.01' or floats like `2.234` anyway I will look up to a solution...

Reply:
Done.. copied from https://github.com/moneyphp/money/blob/master/src/Money.php#L68